stylesheets/_component.buttons.scss

```diff
@@ -316,3 +316,16 @@ $btn-white:                 'white';
         }
     }
 }
+
+.view-history-link {
```

Comment:
This shouldn't be in the generic button component, we should try achieve this either through utility classes, or as a part of the timeline styles

Reply:
I'm afraid it cannot be utility class because they are a bit inflexible regarding responsiveness. Will move them into timeline styles.